### ISSUE TITLE
Unable to run

### ISSUE THREAD

**Issue by tomcelinski** [JKHeadley/appy-backend#13]
Hi,

I'm running node v8.4.0 on OSX 10.12.4. I'm running the latest version of mongodb (v3.4.7). I just installed appy via npm install and get the following:
```

npm WARN deprecated nodemailer@2.7.2: All versions below 4.0.1 of Nodemailer are deprecated. See https://nodemailer.com/status/
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
npm WARN deprecated mongodb@2.2.16: Please upgrade to 2.2.19 or higher
npm WARN deprecated lodash-node@3.6.0: This package is discontinued. Use lodash@^4.0.0.
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.

> fsevents@1.1.2 install /Users/tom/dev/hapi/appy/node_modules/fsevents
> node install

[fsevents] Success: "/Users/tom/dev/hapi/appy/node_modules/fsevents/lib/binding/Release/node-v57-darwin-x64/fse.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN appy@1.0.0 No repository field.
npm WARN appy@1.0.0 No license field.
```
When I run "gulp seed" I get the following:
```

/Users/tom/dev/hapi/appy/node_modules/require-dir/index.js:93
            if (!require.extensions.hasOwnProperty(ext)) {
                                    ^

TypeError: require.extensions.hasOwnProperty is not a function
    at requireDir (/Users/tom/dev/hapi/appy/node_modules/require-dir/index.js:93:37)
    at Object.<anonymous> (/Users/tom/dev/hapi/appy/gulpfile.js:9:23)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
    at Module.load (module.js:507:32)
    at tryModuleLoad (module.js:470:12)
    at Function.Module._load (module.js:462:3)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
    at execute (/usr/local/lib/node_modules/gulp-cli/lib/versioned/^3.7.0/index.js:26:18)
```
I would appreciate any suggestions on this.

Tom

**Comment by isaacmtz90** [comment]
`require.extensions `has been deprecated since node v0.10.6 and upgrading to node 8, which fully drops support for it, causes that error.

You should update on  `package.json` -> ` "require-dir" : "^0.3.2"`. I'm going to send a pull request with the fix

**Comment by JKHeadley** [comment]
Fixed by #14
